ntbs-service/wwwroot/css/_colors.scss

```diff
@@ -24,10 +25,10 @@ $light-greyish-yellow: #DDDACF;
 // Rename colours based on function
 $application-brand: $ukhsa-teal;
 $application-brand-dull: $ukhsa-teal-dull;
-$ref-lab-header: $ukhsa-midnight;
+$ref-lab-header: $ukhsa-moonlight;
 
-$secondary-button-color: $grey;
-$secondary-button-focus-colour: $dark-grey;
+$secondary-button-color: $ukhsa-midnight;
+$secondary-button-focus-colour: $ukhsa-midnight-dull;

```

Comment:
I've commented on the ticket, but I'm not sure this colour is the best choice for secondary buttons.